mavros/launch/apm.launch

```diff
@@ -3,6 +3,7 @@
 	<!-- example launch script for ArduPilotMega based FCU's -->
 
 	<arg name="fcu_url" default="/dev/ttyACM0:57600" />
+    <arg name="fcu_protocol" default="v1.0" />
```

Comment:
indentation

Reply:
And put this and the bellow line under the `log_output` arg so it matches the PX4 launch file.